Tests/Fixtures/Generated/Strings/objc-h/plurals-advanced.h

```diff
@@ -0,0 +1,19 @@
+// Generated using SwiftGen — https://github.com/SwiftGen/SwiftGen
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface LocPluralAdvanced : NSObject
+// multiple.placeholders.string-int --> "Plural case 'other': %@ has %d ratings"
++ (NSString*)multiplePlaceholdersStringIntWithValue:(NSInteger)p1
```

Comment:
ditto obviously: seems like we're missing a `(id)` as first parameter here for the `%@` placeholder?